src/components/pages/single-article/CommentsList.jsx

```diff
@@ -0,0 +1,23 @@
+import { useState, useEffect } from "react"
+import { getCommentsByArticleId } from "../../../../utils/api"
+import CommentCard from "./CommentCard"
+
+export default function CommentsList({article_id}) {
+
+  const [comments, setComments] = useState([])
+
```

Comment:
Please set up a state for loading and let the user know that the comments are loading.